m2cgen/__init__.py

```diff
@@ -11,6 +11,7 @@
     export_to_powershell,
     export_to_r,
     export_to_php,
+    export_to_dart
```

Comment:
```suggestion
    export_to_dart,
```

As this list is supposed to be enhanced in the future, adding comma decreases future diffs and makes review process a little bit easier. Refer to https://stackoverflow.com/a/11597911.